p4runtime_sh/test.py

```diff
@@ -67,6 +70,9 @@ def StreamChannel(self, request_iterator, context):
                 rep.arbitration.CopyFrom(req.arbitration)
                 rep.arbitration.status.code = code_pb2.OK
                 yield rep
+            elif req.HasField('packet'):
+                self.stored_packet_out.put(req)
+                yield p4runtime_pb2.StreamMessageResponse()
```

Comment:
I don't think this yield statement is needed / makes sense. When a P4Runtime server receives a PacketOut message, it doesn't need to reply with an empty stream message response.

Reply:
Right, I think I put this here by mistake